blogs/whats_new_1.7/index.md

```diff
@@ -0,0 +1,153 @@
+- Enhance secondary interface support
+- eBPF Agent enhancements
+- Network Observability in Developer perspective
+- Security features
+- Network Observability CLI enhancements
+
+## Export data using OpenTelemetry Protocol (OTLP)
+
+OpenTelemetry is a popular open source framework for managing telemetry data.  Network Observability can now export flow logs and flow metrics using the OpenTelemetry Protocol (OTLP).
+
+To do this, after you install the Network Observability operator, go to **Operators > Installed Operators**.  On the Network Observability row, click the **Flow Collector** link under the **Provided APIs** column.  Click **Create FlowCollector**.  At the bottom, click **Advanced configuration** to reveal the **Exporters** section (Figure 1).
+
+![Exporters - OpenTelemetry](images/flowcollector-exporters-opentelemetry.png)
+<p style="text-align: center">Figure 1: Exporters - OpenTelemetry</p>
+
+By default, the field names follow the [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/).  The [Network flows format reference](https://docs.openshift.com/container-platform/4.17/observability/network_observability/json-flows-format-reference.html) shows the field mapping in the new **OpenTelemetry** column.  In case you prefer a different name, any of the fields can be renamed to your liking under the **fieldMapping** section (Figure 2).
+
+![OpenTelemetry configuration](images/flowcollector-opentelemetry.png)
+<div style="text-align: center">Figure 2: OpenTelemetry configuration</div>
+
+
+## Enhanced secondary interface support
+
+There are several use cases for wanting a secondary interface on a pod.
+
+1. Partition a NIC in SR-IOV and use the Multus and SR-IOV CNI plugins to attach an SR-IOV virtual function to your pod.  Network Observability supported this in 1.4.
+
+2. Create a [User Defined Network (UDN)](https://docs.openshift.com/container-platform/4.17/networking/multiple_networks/understanding-user-defined-network.html).  UDN provides isolation and segmentation to your traffic, and is a Tech Preview in OCP 4.17.  The pods have a secondary interface to connect to UDN.
+
+3. Create a secondary overlay network in [OpenShift Virtualization](https://docs.openshift.com/container-platform/4.17/virt/about_virt/about-virt.html) to allow east-west traffic between virtual machines (VMs).  The VMs have a secondary interface to connect to the overlay network.
+
+Network Observability 1.7 supports all three scenarios.  Just enable **privileged** mode in eBPF Agent when you create the FlowCollector resource.  This is under the **Agent configuration > eBPF Agent configuration**.  NetObserv handles the rest by tracking the secondary interfaces that come and go, and identifying which namespace and pod they belong to.  In **Observe > Network Traffic**, Traffic flows table, click **Show advanced options** on the right side, then **Manage columns**, and add the column **Interfaces and Directions** (Figure 3).
```

Comment:
I would be good to mention you can configure how these are indexed in the pipeline. See processor.advanced.secondaryNetworks option.

https://docs.openshift.com/container-platform/4.17/observability/network_observability/network-observability-secondary-networks.html

Reply:
I added a link to refer to the documentation.